state/elasticsearch.yaml

```diff
@@ -78,8 +79,13 @@ Parameters:
     Type: String
     Default: '5.5'
     AllowedValues: ['7.7', '7.4', '7.1', '6.8', '6.7', '6.5', '6.4', '6.3', '6.2', '6.0', '5.6', '5.5'] # aws es list-elasticsearch-versions --query "ElasticsearchVersions[]"
+  AvailabilityZones:
```

Comment:
What's the benefit of using a `String` instead of a `Number` type?

Reply:
I can find the configuration value faster in the list of parameters, because it looks differently visually. I do not know anymore why I choose a string instead of a number, because the change was made months ago. It could be that I wanted to create a similarity to the 2azs and 3azs vpc templates.